ibis/backends/duckdb/__init__.py

```diff
@@ -67,6 +112,24 @@ def do_connect(
         )
         self._meta = sa.MetaData(bind=self.con)
 
+    def register(
+        self,
+        file_name: str | Path,
+        table_name: str | None = None,
+    ) -> None:
+        """Register an external file (csv or parquet) as a table in the current
```

Comment:
Probably best to leave off the mention of specific file types in the top level docstring to allow for later expansion without having to update this list. I imagine it will be ever-growing, if slowly.